pkg/operator/ceph/object/spec.go

```diff
@@ -678,3 +669,83 @@ func (c *clusterConfig) rgwTLSSecretType(secretName string) (v1.SecretType, erro
 func getDaemonName(rgwConfig *rgwConfig) string {
 	return fmt.Sprintf("ceph-%s", generateCephXUser(rgwConfig.ResourceName))
 }
+
+func (c *clusterConfig) setRGWSSEKMSDefaultOptions() []string {
```

Comment:
How about we remove the "setRGW" prefix on all these helper methods? It's a bit more readable with fewer acronyms together.
```suggestion
func (c *clusterConfig) sseKMSDefaultOptions() []string {
```